projects/components/src/file-upload/file-upload.component.ts

```diff
@@ -198,4 +205,10 @@ export interface UploaderConfig {
   supportedFileTypes: SupportedFileType[];
   maxFileSizeInBytes: number;
   maxNumberOfFiles: number;
+  customRequirementsInfo?: FileUploadCustomRequirementInfo[];
```

Comment:
Do we want this to be optional? Can't it just be an empty array instead of undefined?

Reply:
We don't want the consumer to always provide the field if they really don't want to use it.

Reply:
Optional fields are easy to miss in case one don't know about it. So that's why

Reply:
No problem